sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala

```diff
@@ -270,12 +270,27 @@ trait CheckAnalysis extends PredicateHelper {
 
         operator match {
           case o if o.children.nonEmpty && o.missingInput.nonEmpty =>
+            val resolver = plan.conf.resolver
+            val attrsWithSameName = o.missingInput.filter(missing =>
+              o.inputSet.exists(input => resolver(missing.name, input.name)))
```

Comment:
could you change the format?
```
o.missingInput.filter { missing =>
  o.inputSet.exists(input => resolver(missing.name, input.name))
}
```

Reply:
Done. Do you know if there is any style rule for when to use parentheses vs braces in these kind of lambda functions?

Reply:
Usually parentheses are used in one line, while curly braces are used in multi-line. For complete scala style guide, please check this: https://github.com/databricks/scala-style-guide

Reply:
Thanks @wzhfy